SYCL/Matrix/joint_matrix_bfloat16_row_major.cpp

```diff
@@ -0,0 +1,170 @@
+//==-------joint_matrix_bfloat16_row_major.cpp  - DPC++ joint_matrix--------==//
```

Comment:
should this be called ...row_majorB?

Reply:
i really has question about naming the testcase here.
SYCL/Matrix/joint_matrix_bfloat16_col_majorA.cpp(A is colmajor and B is colmajor, we can test colmajor=>packed and colmajor=>rowmajor)
i suggest change it into joint_matrix_bfloat16_colmajorA_colmajorB.cpp
joint_matrix_bfloat16_row_major.cpp => joint_matrix_bfloat16_rowmajorA_rowmajorB.cpp(we can test colmajor=>packed)
with joint_matrix_bfloat16_colmajorA_colmajorB.cpp and  joint_matrix_bfloat16_rowmajorA_rowmajorB.cpp, we can test all the auto-vnnitransform cases.

Reply:
I agree with the naming you have now